CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Not released
 
+- Update deck.gl version to 8.5.0 [#264](https://github.com/CartoDB/carto-react-template/pull/264)
+- Fix external libraries warnings [#264](https://github.com/CartoDB/carto-react-template/pull/264)
```

Comment:
If you're gonna leave an independent line here, then try to be more explicit. Eg: "Fix material-ui warnings on justify property"

Reply:
Okey